src/etc/inc/services.inc

```diff
@@ -763,7 +775,15 @@ EOPP;
 				$dhcpdconf .= '}' . "\n";
 			}
 		}
-
+		if (isset($config['dhcpd'][$dhcpif]['enable_aliases'])) {
+			$dhcpdconf .= "shared-network {$dhcpif} {\n";
+			foreach ($vipaliases as $vipalias) {
+				$vipsubnet = gen_subnet($vipalias['subnet'], $vipalias['subnet_bits']);
+				$vipsubnetbits = $vipalias['subnet_bits'];
+				$vipsubnetmask = gen_subnet_mask($vipalias['subnet_bits']);
+				$dhcpdconf .= "subnet {$vipsubnet} netmask {$vipsubnetmask} {}\n";
```

Comment:
You also need to consider failover here. Though it is only being used for static addresses in this setup, it should probably still have the failover options present. Needs testing, though.